test/e2e_node/oomkiller_linux_test.go

```diff
@@ -37,6 +38,26 @@ type testCase struct {
 	oomTargetContainerName string
 }
 
+const KubeReservedMemory = "1500Mi"
+
+var _ = SIGDescribe("ndixita OOMKiller [LinuxOnly] [Serial]", func() {
+	f := framework.NewDefaultFramework("nodeallocatable-oomkiller-test")
```

Comment:
As the test intends to use up the entire node memory, and worth to mention what machine type is it target for ? As for serial node e2e, it uses `n1-standard-2`, so it should be good, but in case if the machine type changes in future.

Reply:
It should run the serial test on any machine type? I have added the [Serial] tag for it. Please let me know if my understanding here is incorrect.